docs/advanced/custom-map-controls.md

```diff
@@ -35,18 +42,28 @@ A custom map controls class must implement the following interface:
 
 ##### `events` (Array)
 
-A list of event names that this control subscribes to.
+A list of additional event names that this control subscribes to.
 
-Available events: `click`, `tap`, `doubletap`, `press`, `pinch`, `pinchin`, `pinchout`, `pinchstart`, `pinchmove`, `pinchend`, `pinchcancel`, `rotate`, `rotatestart`, `rotatemove`, `rotateend`, `rotatecancel`, `pan`, `panstart`, `panmove`, `panup`, `pandown`, `panleft`, `panright`, `panend`, `pancancel`, `swipe`, `swipeleft`, `swiperight`, `swipeup`, `swipedown`, `pointerdown`, `pointermove`, `pointerup`, `touchstart`, `touchmove`, `touchend`, `mousedown`, `mousemove`, and `mouseup`.
+Available events: `click`, `tap`, `doubletap`, `press`, `pinch`, `pinchin`, `pinchout`, `pinchstart`, `pinchmove`, `pinchend`, `pinchcancel`, `rotate`, `rotatestart`, `rotatemove`, `rotateend`, `rotatecancel`, `pan`, `panstart`, `panmove`, `panup`, `pandown`, `panleft`, `panright`, `panend`, `pancancel`, `swipe`, `swipeleft`, `swiperight`, `swipeup`, `swipedown`, `pointerdown`, `pointermove`, `pointerup`, `touchstart`, `touchmove`, `touchend`, `mousedown`, `mousemove`, `mouseup`, `keydown`, and `keyup`.
 
-[Event object](http://hammerjs.github.io/api/#event-object) is generated by [hammer.js](http://hammerjs.github.io).
+The following events are toggled on/off by InteractiveMap props: 
 
-### Methods
+- `scrollZoom` - `['wheel']`
+- `dragPan` and `dragRotate` - `['panstart', 'panmove', 'panend']`
+- `touchZoomRotate` - `['pinchstart', 'pinchmove', 'pinchend']`
+- `doubleClickZoom` - `['doubletap']`
+- `keyboar` - `['keydown']`
 
-##### `setState(state)`
+Event object is generated by [mjolnir.js](https://github.com/uber-web/mjolnir.js/blob/master/docs/api-reference/event-manager.md).
```

Comment:
FYI I still can't access the `mjolnir.js` repo :( - trying to follow up to see if it's already public or not.

Reply:
Good call. I'll copy it over for now.